x-pack/plugins/security_solution/cypress/screens/timeline.ts

```diff
@@ -63,7 +63,7 @@ export const NOTE_CONTENT = (noteId: string) => `${NOTE_BY_NOTE_ID(noteId)} p`;
 
 export const NOTES_TEXT_AREA = '[data-test-subj="add-a-note"] textarea';
 
-export const NOTES_TAB_BUTTON = '[data-test-subj="timelineTabs-notes"]';
+export const NOTES_TAB_BUTTON = 'button[data-test-subj="timelineTabs-notes"]';
```

Comment:
Is the button addition needed here to add more specificity? Usually if I can get away with just a pure data-test-subj, I will use just that so tests don't start failing if the HTML type changes from button to something else.

Reply:
I meant to remove that after I checked one of the ones you'd done and saw you hadn't specified. Thanks for pointing that out, I'll open a PR to cleanup.